src/pe/mod.rs

```diff
@@ -410,7 +410,7 @@ impl<'a> ctx::TryIntoCtx<scroll::Endian> for PE<'a> {
             }
             _ => None,
         };
-        bytes.gwrite_with(self.header, &mut offset, ctx)?;
+        bytes.gwrite_with(self.header.clone(), &mut offset, ctx)?;
```

Comment:
i think keeping copy derive on self.header is fine, and also its a breaking change to remove

Reply:
Ah, I agreeed. there was a borrow[^1] of `self.header` after this code and I think I was too lazy and let it clone, but it shouldn't be.

[^1]: [`"Symbol tables in PE are deprecated and not supported to write"` assertion](https://github.com/m4b/goblin/pull/406/files#diff-6303a321bf22fd69f034c1622e2b63c543fb1ceb6f3828e3dd81403e8fff4942R431)

Reply:
I guess solved in 7bcc51b0b83a44d94b051f3079c8dac5fc8c3b20.
I simply moved the assertion before the `bytes.gwrite_with(self.header, &mut offset, ctx)?;`, since nothing modifies `self.header.coff_header.pointer_to_symbol_table` betweeen the grwrite and the assertion, so it's semantically the same no no problems.
Please let me know if you have better option. Otherwise I'd like you to mark this change request as resolved in your end.